Alpha_wrap_3/doc/Alpha_wrap_3/alpha_wrap_3.txt

```diff
@@ -0,0 +1,370 @@
+namespace CGAL {
+
+/*!
+
+\mainpage User Manual
+\anchor Chapter_3D_Alpha_wrapping
+\cgalAutoToc
+
+\authors TBA
+
+<center>
+<img src="aw3_bike_lod.png" style="max-width:70%;"/>
+</center>
+
+\section aw3_introduction Introduction
+
+Various tasks in geometric modeling and processing require 3D objects represented as valid surface meshes,
+where "valid" refers to meshes that are watertight, intersection-free, orientable, and 2-manifold.
+Such representations offer well-defined notions of interior/exterior and geodesic neighborhoods.
+
+3D data are usually acquired through measurements followed by reconstruction, designed by humans,
+or generated through imperfect automated processes.
+As a result, they can exhibit a wide variety of defects including gaps, missing data,
+self-intersections, degeneracies such as zero-volume structures, and non-manifold features.
+
+Given the large repertoire of possible defects, many methods and data structures have been proposed
+to repair specific defects, usually with the goal of guaranteeing specific properties in the repaired 3D model.
+Reliably repairing all types of defects is notoriously difficult and is often an ill-posed problem
+as many valid solutions exist for a given 3D model with defects.
+In addition, the input model can be overly complex with unnecessary geometric details,
+spurious topological structures, nonessential inner components, or excessively fine discretizations.
+For applications such as collision avoidance, path planning, or simulation,
+getting an approximation of the input can be more relevant than repairing it.
+Approximation herein refers to an approach capable of filtering out inner structures,
+fine details and cavities, as well as wrapping the input within a user-defined offset margin.
+
+Given an input 3D geometry, we address the problem of computing a conservative approximation,
+where conservative means that the output is guaranteed to strictly enclose the input.
+We seek unconditional robustness in the sense that the output mesh should be valid (oriented,
+2-manifold, and without self-intersections), even for raw input with many defects
+and degeneracies.
+The default input is a soup of 3D triangles, but the generic interface leaves the door open
```

Comment:
should be updated as points and segments are handled, right?

Reply:
Segments API is fully undocumented and I don't want to put emphasis on the Points interface.